homeworks/andrii.kholodenko_AndriiCold/6-typescript/src/app.ts

```diff
@@ -0,0 +1,103 @@
+import './styles/style.css';
+
+const totalScoreEl = document.querySelector('[data-total-score]') as HTMLHeadingElement;
+const bubbleEl = document.querySelector('[data-bubble]') as HTMLDivElement;
+const dynamicScoreEl = document.querySelector('[data-dynamic-score]') as HTMLDivElement;
+const keyEl = document.querySelector('[data-key]') as HTMLDivElement;
+
+let score: number = 100;
+let currentKey: string = '';
+const interval: number = 2000;
+let randomNumber: number;
+let timer: NodeJS.Timeout;
+
+function updateRandomValue(value: string): void {
+    keyEl.innerText = value;
+}
+
+function updateScore(score: string): void {
+    totalScoreEl.innerText = score;
+}
+
+function updateBubbleScore(score: string): void {
+    dynamicScoreEl.innerText = score;
+}
+
+function bubbleSize(): void {
+    bubbleEl.style.width = String(score + 50 + 'px');
+    bubbleEl.style.height = String(score + 50 + 'px');
+}
+
+function setKey(): string {
+    const letter: string = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ';
```

Comment:
```suggestion
    const letter: alphabet = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ';
```
Тут не одна літера, а алфавіт